api_app/analyzers_manager/file_analyzers/droidlysis.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from api_app.analyzers_manager.classes import DockerBasedAnalyzer, FileAnalyzer
+from tests.mock_utils import MockUpResponse
+
+logger = logging.getLogger(__name__)
+
+
+class DroidLysis(FileAnalyzer, DockerBasedAnalyzer):
+    name: str = "android_analyzer"
+    url: str = "http://malware_tools_analyzers:4002/droidlysis"
+    # interval between http request polling
+    poll_distance: int = 2
+    # http request polling max number of tries
+    max_tries: int = 5
+
+    def update(self) -> bool:
+        pass
+
+    def run(self):
+        binary = self.read_file_bytes()
+        fname = str(self.filename).replace("/", "_").replace(" ", "_")
+        args = [
+            "--input",
+            f"@{fname}",
+            "-o",
+            "/opt/deploy/droidlysis/out",
```

Comment:
if you do that, which type of output are they creating? JSON? output for the CLI? can you test it?
It needs not only to be JSON but also to be pushed as stdout. Otherwise you would need an interceptor (see app.py) and extract the data from the file.

Reply:
that makes sense, but right the tool itself can't be used inside the container as you maybe able to see in the error image. The Pr is draft and there are many rough edges to it. The main problem I'm facing right now is the current error. Do you think it might be related to any sort of permissions?

Reply:
yes, this line: https://github.com/cryptax/droidlysis/blob/master/droidconfig.py#L55 the code is public so it's possible to see what's happening behind the curtains. I think it could be enough to create this file beforehand in the dockerfile with the proper permissions

Reply:
Understood!